ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -650,3 +650,33 @@ def verify_odf_without_ocs_in_installed_operator(self) -> bool:
             expected_text="OpenShift Container Storage",
         )
         return odf_operator_presence and not ocs_operator_presence
+
+    def verify_storage_clients_page(self):
```

Comment:
I think this method should be split in two, because the name says verify_storage_clients_page but it returns onboarding key. 
We need to have a method that will do only get_onboarding_key